web-animations-1/Overview.bs

```diff
@@ -708,6 +701,35 @@ animation model to change, but these operations in themselves do not create
 a new [=animation frame=], rather they merely update the current <a>animation
 frame</a>.
 
+### Timeline Phase ### {#timeline-phases}
+
+A [=timeline=] may be in one of four possible 
+<dfn export lt="timeline phase">phases</dfn> for each of which, a non-normative
+description is also provided:
+
+<div class=informative-bg>
+
+:   <dfn export lt="timeline inactive phase">inactive</dfn>
+::  A <a>timeline</a> is considered to be <dfn lt="inactive timeline">inactive
+    </dfn> when its <a>time value</a> is <a>unresolved</a>. This behavior is
+    further defined by each type of timeline.
+:   <dfn export lt="timeline before phase">before</dfn>
+::  The timeline current time is before its start point.
+:   <dfn export lt="timeline active phase">active</dfn>
+::  The timeline current time is after its start point and before its end point.
+    Note: There is a special case for ScrollTimeline that can make the end point
+    inclusive.
+:   <dfn export lt="timeline after phase">after</dfn>
+::  The timeline current time is at or beyond its end point.
```

Comment:
I don't think this makes sense to have these as exported definitions that are non-normative. I think if we are linking to these we should at least just list them in normative text. We can then follow up with a non-normative section describing them in more detail.